src/utils/requiredSignoffs.js

```diff
@@ -0,0 +1,21 @@
+export default (rule, rs) => {
```

Comment:
Thoughts on renaming this file to `ruleMatchesRequiredSignoff`?

Reply:
I was thinking of this file as a place to keep general helpers around processing/evaluating required signoffs, which is why I named it this way. Maybe I should export a specific named function rather than this one as the default? What do you think?

Reply:
Exporting a specific named function makes a lot of sense.